features/steps/topic_component.go

```diff
@@ -38,39 +39,20 @@ func NewTopicComponent(mongoFeature *componenttest.MongoFeature, zebedeeURL stri
 	}
 
 	var err error
-
 	f.Config, err = config.Get()
 	if err != nil {
 		return nil, err
 	}
 
-	f.Config.ZebedeeURL = zebedeeURL
-
-	f.Config.EnablePrivateEndpoints = false // for component tests, ensure 'false' to start
-
-	f.Config.EnablePermissionsAuth = false
-
-	getMongoURI := fmt.Sprintf("localhost:%d", mongoFeature.Server.Port())
-	databaseName := utils.RandomDatabase()
+	f.Config.ClusterEndpoint, f.Config.ZebedeeURL, f.Config.Database, f.Config.EnablePrivateEndpoints =
+		mongoURL, zebedeeURL, utils.RandomDatabase(), false
```

Comment:
Isn't it more readable if we had one assignment per line?

Reply:
Done